### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by nii236**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
```
Comment (nii236) [inline readme.md:2]: This looks good

**Comment by nii236** [comment]
I dunno... some spelling mistakes?

**Comment by nii236** [comment]
Seems ok

**Comment by AlexDunmow** [comment]
Hi there

**Comment by AlexDunmow** [comment]
Woop

**Pull request merged**